src/demo-app/menu/menu-demo.html

```diff
@@ -140,7 +140,7 @@
     <p>overlapTrigger: true</p>
 
     <mat-toolbar>
-      <button mat-icon-button [mat-menu-trigger-for]="menuOverlay">
```

Comment:
It's a bummer that these deprecated attributes stayed in the demos for all of v6 after being deprecated in v5 😦 

Can this be patched back to the v6 branch?

Reply:
Nevermind, this is just in the demo-app.